README.md

```diff
@@ -1,24 +1,170 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# 📗 Table of Contents
 
-Things you may want to cover:
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [🙏 Copyright](#copyrights)
+- [📝 License](#license)
 
-* Ruby version
+<!-- PROJECT DESCRIPTION -->
 
-* System dependencies
+# 📖 RECIPE APPLICATION <a name="about-project"></a>
 
-* Configuration
+**HELLO RAILS REACT APPLICATION** is a educational application where a connection between a Ruby on Rails back-end and React front-end is implemented.
 
-* Database creation
+## 🛠 Built With <a name="built-with"></a>
 
-* Database initialization
+### Tech Stack <a name="tech-stack"></a>
 
-* How to run the test suite
+<details>
+  <summary>Editor</summary>
+  <ul>
+    <li><a href="https://code.visualstudio.com/">VSCODE Editor</a></li>
+  </ul>
+</details>
+<details>
+  <summary>Backend</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+<details>
+<summary>FrontEnd</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">REACT</a></li>
+  </ul>
+</details>
 
-* Services (job queues, cache servers, search engines, etc.)
+### Key Features <a name="key-features"></a>
 
-* Deployment instructions
+- **A random greeting is displayed**
 
-* ...
```

Comment:
- [ ] Kindly make sure your two apps work together as required for the project. Right now, I can create new greetings and  and when running the API route every thing work as expected but nothing can be displayed on the UI.
![hello](https://user-images.githubusercontent.com/83042742/221090876-b5464946-5842-4f58-83c0-4197fb145e84.gif)